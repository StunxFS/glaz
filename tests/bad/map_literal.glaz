fn main() {
	_ = map<str, i32>{};
}
