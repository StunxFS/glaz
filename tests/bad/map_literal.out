tests/bad/map_literal.glaz:2:7: error: invalid map literal
     | 
   2 | 	_ = map<str, i32>{};
     | 	    ^^^^^^^^^^^^^
     = note: 1 or more key-values are expected
     = help: use `map<str, T>::new()` instead
glaz error: could not compile `map_literal`, aborting due to previous error.
